src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIDE_LENGTH = 100;
+    private static final int MAX_RADIUS = 50;
+    private static final int MAX_TRIANGLE_LEG = 100;
+    private static final int MAX_TRAPEZOID_BASE = 100;
+    private static final int MAX_TRAPEZOID_HEIGHT = 50;
+    private static final Random RANDOM = new Random();
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    private static final int MAX_SIDE = 100;
    private final Random RANDOM = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier();
```

leave only one MAX_SIDE variable